File: INICIANTE/JAVASCRIPT/1117.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) { return lines.shift(); };

for(nv = 0;nv != 1;){
    var x = parseFloat(prompt(""))
    if(x >= 0 && x <= 10){
        nv = nv + 1
    }
    else{
        console.log("nota invalida")
    }
 }
 for(nv = 0;nv != 1;){
     var y = parseFloat(prompt(""))
    if(y >= 0 && y <= 10){
        nv = nv + 1
    }
    else{
        console.log("nota invalida")
    }
 }
 var media = (x + y) / 2
 console.log("media = " + media.toFixed(2))

File: INICIANTE/JAVA/1117.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        float x,media = 0,soma = 0,nv = 0;
        Scanner sc = new Scanner(System.in);
        
        while(nv != 2){
            x = sc.nextFloat();
            if(x >= 0 && x <= 10){
                nv = nv + 1;
                soma = soma + x;
            }
            else{
                System.out.println("nota invalida");
            }
        }
        media = (float) soma / 2;
        System.out.println(String.format("media = %.2f",media));
        sc.close();
    }
}
